pkg/pki/x509/x509.go

```diff
@@ -81,12 +82,12 @@ func (s Signature) Verify(r io.Reader, k interface{}) error {
 		if ed25519.Verify(pub, message, s.signature) {
 			return nil
 		}
-		return fmt.Errorf("signature mismatch for %s", string(s.signature))
+		return errors.New("supplied signature does not match key")
 	case *ecdsa.PublicKey:
 		if ecdsa.VerifyASN1(pub, hash, s.signature) {
 			return nil
 		}
-		return fmt.Errorf("signature mismatch for %s", string(s.signature))
+		rreturn errors.New("supplied signature does not match key")
```

Comment:
you have an extra 'r' on your return there

Reply:
doh! fixed